starter-code/app.js

```diff
@@ -7,6 +7,11 @@ const mongoose       = require("mongoose");
 const app            = express();
```

Comment:
missing require and setup of the two node modules needed to manage and store sessions:

- `express-session` - middleware to manage browser sessions
- `connect-mongo` - extension to store those sessions in our mongo database

```
const session = require("express-session");
const MongoStore = require("connect-mongo")(session);

app.use(session({
  secret: "basic-auth-secret",
  cookie: { maxAge: 60000 },
  store: new MongoStore({
    mongooseConnection: mongoose.connection,
    ttl: 24 * 60 * 60 // 1 day
  }),
  resave: true,
   saveUninitialized: true
 }));
```

without these Express will not send/read the express session cookie that allows our app to identify which user is behind each browser's requests